### PR TITLE
Add beta /ci setup instructions

### PR DESCRIPTION
**Pull request by ransombriggs**
@mathias could you review?

### CODE REVIEW

File: app/controllers/setup_controller.rb [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
     end
   end
+
+  def index
+
+  end
+
+  def ci
+
+  end
```
Comment (mathias) [inline app/controllers/setup_controller.rb]: No need for empty action methods here, but it doesn't hurt, either.

File: app/views/setup/ci.html.erb [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      <p class="f4 dark-gray lh-copy">
+        Replace $TRIGGER_PIPELINE with the name or id of the pipeline on which you have tests enabled.
+      </p>
+      <% if Rails.env.development? %>
+        <p class="f4 dark-gray">
+          Replace $PUBLIC_HOST with the publicly addressable host of your development server
+        </p>
+      <% end %>
+      <div class="code-block relative lh-copy mv3 w-100 bg-light-silver  code pa2 pl5 dark-gray ba b--silver br1 word-wrap">
+        <div>heroku webhooks:add --include api:test-run -l sync --url <%= @webhooks %> -s <%= @webhook_secret %> -p $TRIGGER_PIPELINE</div>
```
Comment (mathias) [inline app/views/setup/ci.html.erb]: `@webhooks` is a string value here?

Comment (mathias) [reply app/views/setup/ci.html.erb]: (And `webhook-secret`?)

**Comment by mathias** [comment]
@ransombriggs Some questions above.

File: app/controllers/setup_controller.rb (hunk 1 above)
Comment (ransombriggs) [reply app/controllers/setup_controller.rb]: @mathias took out in https://github.com/heroku/webhooks-demo/pull/30/commits/0e20daa39db5513ed540e4eb8997c59f3b234eb9

File: app/views/setup/ci.html.erb (hunk 2 above)
Comment (ransombriggs) [reply app/views/setup/ci.html.erb]: @mathias yes, from here https://github.com/heroku/webhooks-demo/pull/30/files#diff-96d7538446a97258fe291a027a04c6baR3

**Pull request merged**